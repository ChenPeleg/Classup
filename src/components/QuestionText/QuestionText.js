import React from "react";
import styles from "./QuestionText.module.scss";
const QuestionText = (props) => {
  return (
    <>
      <div className={styles.centeringDiv}>
        <pre className={styles.questionText}> {props.children}</pre>
      </div>
    </>
  );
};
export default QuestionText;
