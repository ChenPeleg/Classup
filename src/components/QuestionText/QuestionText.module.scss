@import "../../assets/styles/base";
.img_0 {
  position: relative;
  display: inline-block;
  height: 330px;
  width: 80%;
  height: 70%;
}
.questionText {
  position: relative;
  font-family: $Base_fontFace;

  white-space: pre-line;
}
.centeringDiv {
  padding-left: 100px;
  padding-right: 100px;
  text-align: left;
  padding: auto;
}
