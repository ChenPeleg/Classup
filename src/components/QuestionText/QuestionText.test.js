import React from "react"
import { shallow } from "enzyme"
import QuestionText from "./QuestionText"

const wrap = shallow(<QuestionText />)

describe("<QuestionText>", () => {
    it("renders correctly", () => {
        expect(wrap).toMatchSnapshot()
    })
})