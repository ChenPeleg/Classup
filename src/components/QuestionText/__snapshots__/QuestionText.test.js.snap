// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<QuestionText> renders correctly 1`] = `ShallowWrapper {}`;
